vvvv45/addonpack/src/nodes/plugins/Animation/DelayFramebased/DelayFramebasedNode.cs

```diff
@@ -0,0 +1,117 @@
+#region usings
+using System;
+using System.ComponentModel.Composition;
+
+using System.Collections.Generic;
+using System.Collections;
+
+using VVVV.PluginInterfaces.V1;
+using VVVV.PluginInterfaces.V2;
+using VVVV.Utils.VColor;
+using VVVV.Utils.VMath;
+
+using VVVV.Core.Logging;
+#endregion usings
+
+namespace VVVV.Nodes
+{
+	#region PluginInfo
+	[PluginInfo(Name = "Delay", Category = "Animation", Version ="Framebased", Help = "Framebased delay with adjustable frame count")]
+	#endregion PluginInfo
+	public class DelayFramebased : IPluginEvaluate
+	{
+		#region fields & pins
+		[Input("Input", DefaultValue = 1.0)]
```

Comment:
any reason for the default being 1.0 here instead of 0? if not, then simply remove the default value